data/core/units/wose/Ancient.cfg

```diff
@@ -14,7 +14,7 @@
     advances_to=null
     {AMLA_DEFAULT}
     cost=48
-    description= _ "There is a curious story of a farmer who had a great oak in his lawn, a tree which had stood when his father first broke the soil on that land, and under which his family had many memories. It was thus a shock to him, one morning, when he awoke to find it somehow spirited away, with nothing but a bare patch of earth where it once stood.
+    description= _ "There is a curious story of a farmer who had a great oak in his lawn, a tree which had stood when his father first broke the soil on that land, and under which his family had many memories. It was thus a shock to him, one morning, when he awoke to find it had somehow spirited away, with nothing but a bare patch of earth where it once stood.
```

Comment:
I'm not sure if this change is actually correct? "to find it spirited away" is a valid phrase, I believe, though it does imply something had taken it, rather than it taking off on its own.

Reply:
Or it might mean the wose got itself inebriated. :smile: 

I assume the wose was sentient and took off under its own steam.

Reply:
Indeed.

You might be able to argue that the storyteller didn't know this, however, and thus used the term simply to indicate it had mysteriously disappeared.

Reply:
I think that part is spoken by an [omniscient narrator](https://en.wikipedia.org/wiki/Narration#Third-person,_omniscient), not by the farmer. But I see there's more than one way to parse this.

Reply:
The way I was reading it is a scholar reporting a story he heard. So the narrator is not the farmer, but not omniscient either.

But this is the in-game help, so it could be an omniscient narrator.

Reply:
I see. How about this, then?

"It was thus a shock to him, one morning, when he awoke to find ~it somehow spirited away, with~ nothing but a bare patch of earth where ~it~ the tree once stood, as though it had spirited away in the wee hours of the night."

Reply:
I just figured out the oddness of your proposals using "had spirited away"... the phrase is _transitive_, which means it shouldn't be used without an object. If you want to make the tree's agency clearer, I think a different phrase might be needed...

To clarify:

* "to find it somehow spirited away" - Here, it's used as a past participle rather than a full verb, and it's modifying "it", which fills the role of the object/patient of the verb. Of course, this implies something did the spiriting away.
* "to find it had somehow spirited away" - Here, it's used as a verb in past perfect (so there's still a past participle, but it's part of a conjugated verb rather than standing alone as a gerundive). There's no object/patient, so it sounds strange.
* "to find it had been somehow spirited away" - There's no object here, but it's now in passive tense, so the subject ("it") fills the role of the patient. As with the first case, this takes the agency away from the wose, implying something did the spiriting away.

(And "to find it" can be replaced by "as though it" without changing the analysis.)

So unless you want to make it reflexive ("had spirited itself away" for example), I think the phrase "spirited away" can't be used while making the wose's agency in the action explicit.

Reply:
Ah! I thought it was usable in an intransitive sense, equivalent to "had left". Thanks for the thorough analysis/clarification. I agree we should use a phrasing that lets the wose be the agent. How about this? In conjunction with the (unchanged) next paragraph of the description, the wose's agency is implied as a possibility.

> There is a curious story of a farmer who had a great oak in his lawn, a tree which had stood when his father first broke the soil on that land, and under which his family had many memories. It was thus a shock to him, one morning, when he awoke to find nothing but a bare patch of earth where the tree once stood, as though it had been spirited away in the wee hours of the night.
>
> Such encounters are all that is known of what are presumably the most ancient of woses.

Reply:
I have no objections to that.

Reply:
Pushed. I think it's merge ready now, or should I remove that "to" from Masked Dwarvish Lord first?